classes/fbvuln-manifest.bbclass

```diff
@@ -68,7 +68,7 @@
 #   bitbake core-image-minimal
 CVE_PRODUCT ??= "${BPN}"
 CVE_VERSION ??= "${PV}"
-CVE_CHECK_WHITELIST ??= ""
+CVE_CHECK_IGNORE ??= ""
```

Comment:
This will break compatibility.  You could consider the following to maintain the ability to run against older releases.

  CVE_CHECK_IGNORE ??= "${CVE_CHECK_WHITELIST}"

You'll need to test that, but looking at https://docs.yoctoproject.org/migration-guides/migration-3.5.html I think it'll do the trick.

Reply:
I did the backward compatibility test for two past builds plus current Head of helium. This with the change you suggested. Check our internal task for details to build jobs.